src/Directory.build.props

```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Xunit.StaFact" Version="0.3.18" />
     <PackageReference Include="Shouldly" Version=" 3.0.2" />
     <PackageReference Include="ReactiveUI.Testing" Version="11.*" />
-    <PackageReference Include="PublicApiGenerator" Version="9.3.0" />
+    <PackageReference Include="PublicApiGenerator" Version="10.*" />
```

Comment:
Since it's a less controlled resource let's keep it at specific version for security. Eg I just let our own dependencies or Microsoft ones generally he *